lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -34,11 +40,16 @@
                 },
 
                 postRender: function(){
-                    this.enrollView = new CourseEnrollView({
-                        $el: this.$('.course-actions'),
-                        model: this.model,
-                        context: this.context
-                    });
+                    HtmlUtils.setHtml(
+                        this.$('.course-actions'),
+                        HtmlUtils.HTML(
+                            new CourseEnrollView({
+                                model: this.model,
+                                context: this.context,
+                                enrollModel: this.enrollModel
+                            }).$el
+                        )
+                    );
```

Comment:
There is a specific reason I made this change. With passing in the DOM element (the old code), the event listeners defined in the `CourseEnrollView` are actually all overriden. No events will be triggered with the old code. Let me know if you have a good solution in that case.